src/main/java/sx/blah/discord/modules/ModuleLoader.java

```diff
@@ -153,7 +153,7 @@ public void unloadModule(IModule module) {
 			Class<? extends IModule> clazz = module.getClass();
 			if (clazz.isAnnotationPresent(Requires.class)) {
 				Requires annotation = clazz.getAnnotation(Requires.class);
-				if (annotation.value().equals(module.getClass().getName())) {
+				if (annotation.value().toLowerCase(Locale.ROOT).equals(module.getClass().getName())) {
```

Comment:
I don't think the toLowerCase is required because class names are case sensitive.